### DISCUSSION TITLE
开启 Rime/小狼毫 中文输入后网页不能使用快捷键，这能改吗?

### DISCUSSION THREAD

**Discussion by quantumfallout** [rime/home#1373] [General]
在开启中文输入的情况下，一些网页快捷键(例如 YouTube 播放器快捷键，Bilibili 播放器快捷键，Vimium 的快捷键)会无法使用。似乎所有的输入都被小狼毫吃掉了，像微软拼音并不会有这样的情况，请问有什么选项能改么?

例如在YouTube的评论区输入`w`，这时会将此前的按键都输入到评论区文本框中。

https://github.com/rime/home/assets/16105768/606c65f7-0708-4500-8b53-fc768fce948a

**Comment by vkagari** [comment]
我也遇到了这个问题，请问有解决方法吗

**Comment by quantumfallout** [comment]
rime/weasel/issues/997
